environment.yml

```diff
@@ -0,0 +1,14 @@
+channels:
+  - ome
+  - conda-forge
+  - defaults
+dependencies:
+  - bioformats2raw
+  - python == 3.9.0
+  - wheel
+  - s3fs
+  - zarr >= 2.7.0
+  - pip
+  - pip:
+      - ome-zarr
+      - pytest
```

Comment:
any reason not to install pytest via conda i.e. https://anaconda.org/anaconda/pytest
I am still thinking if we want to have a conda.lock at some point, mixing pip and conda implies that the deployment become a 2 steps process.

Reply:
Happy to move it up, but I wouldn't pin here since we _want_ it to fail. We could have one of each though.